.gitignore

```diff
@@ -175,4 +175,4 @@ tags
 .ruff_cache
 
 # wandb
-wandb
+wandb
```

Comment:
Why is this needed?

Reply:
Thank you for your reply.
`wandb` should not have been added by me, it already exists in diffusers.

![image](https://github.com/huggingface/diffusers/assets/38418898/2e7a31a6-c77c-47b6-98b0-3cea775e30af)